site/scripts/library.js

```diff
@@ -72,10 +72,13 @@ export function initializeLibrary() {
 
       /////////////////////////////
       //const editCategoryOptions = document.getElementById("edit-category-options");
-      const option = document.createElement("option");
-      option.value = category.order;
-      option.textContent = category.name;
-      editCategoryOptions.appendChild(option);
+      if(category.order !== -1 && category.order !== 65535)
```

Comment:
Slightly less duplication
```suggestion
      if(![-1, 65535].includes(category.order))
```